locales/en.default.schema.json

```
@@ -1243,6 +1249,9 @@
               "label": "Page"
             }
           }
+        },
+        "rating": {
+          "name": "Rating"
```

```suggestion
          "name": "Product rating"
```
As per Gretchen's recommendation 🙏